package.json

```diff
@@ -32,7 +32,7 @@
     "tape": "^4.5.1"
   },
   "dependencies": {
-    "google-auth-library": "0.9.8",
+    "@google/cloud-diagnostics-common": "0.2.3",
     "lodash": "4.13.1"
```

Comment:
nit: I think it is more important to be able to pickup bug fixes in a timely manner, so it would be better to switch to `^4.13.1` as the dependency for lodash.